ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactHorizontalScrollView.java

```diff
@@ -92,6 +92,10 @@ public void setPagingEnabled(boolean pagingEnabled) {
     mPagingEnabled = pagingEnabled;
   }
 
+  public boolean flashScrollIndicators() {
+    return super.awakenScrollBars();
```

Comment:
Just a small left-over piece of code; I was initially overriding to just widen the scope:

```java
@Override
public boolean awakenScrollBars() {
  return super.awakeScrolBars();
}
```

But I decided to align the names with the existing API; And now the `super` was out of place. Deleted now, and the PR has been updated :)